lobby.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Lobby</title>
+		<link href="https://fonts.googleapis.com/css?family=Inconsolata:700" rel="stylesheet">
+		<link href='http://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
+		<link href="https://fonts.googleapis.com/css?family=Hammersmith+One" rel="stylesheet" type="text/css" />
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<link rel="stylesheet" type="text/css" href="css/basic.css">
+		<link rel="stylesheet" type="text/css" href="css/image-icons.css">
+	</head>
+
+	<body>
+		<header class="header">
+			<nav class="navbar">
+				<a class="navbar_brand" href="#">
+					<span class="text_red">Mono</span>Banker
+				</a>
+			</nav>
+		</header>
+		<div class="content container">
+			<div class="lobby_section">
+				<div class="lobby_section__header">
```

Comment:
Header must be wrapped in `h` element

Reply:
In my case i use only the `<a>` link with different colors. I found a solution using `<span>`. I checked the basic examples on getbootstrap.com (https://getbootstrap.com/docs/4.4/examples/navbars/). They put `.navbar-brand` in `<a>`, and i used this method.

Please share an example with other cases.

Reply:
@dtalkachou I am not actually following. What do you mean?

Reply:
> Header must be wrapped in `h` element

Please attach link to the site where this method was used. Or just attach your code with my situation. I just don’t understand what and why to wrap in the <h> tag.

Reply:
This block of code represents a header of the page. Speaking in terms of semantic markup, you have to use a suitable tag that has following meaning: 'this is a header'. A collection of `h1, h2, h3, etc` tags does exactly this. I want you to use one of these tags instead of plain `div`.